source/postprocess/basic_statistics.cc

```diff
@@ -44,6 +44,9 @@ namespace aspect
               const MaterialModel::Simple<dim> &material_model
                 = dynamic_cast<const MaterialModel::Simple<dim> &>(this->get_material_model());
 
+              MaterialModel::MaterialModelInputs<dim> in (1, this->n_compositional_fields());
+              MaterialModel::MaterialModelOutputs out (1, this->n_compositional_fields());
```

Comment:
what's the purpose of these changes here? They have nothing to do with the simple material model, right?